src/java/src/test/java/ai/onnxruntime/genai/GenAITestExecutionListener.java

```diff
@@ -0,0 +1,10 @@
+package ai.onnxruntime.genai;
+
+import org.junit.platform.launcher.TestExecutionListener;
+import org.junit.platform.launcher.TestPlan;
+
+public class GenAITestExecutionListener implements TestExecutionListener {
+  public void testPlanExecutionFinished(TestPlan testPlan) {
+    GenAI.shutdown();
```

Comment:
Global shutdown is needed. Otherwise leaded resources complain

Reply:
Does the GenAI documentation call this requirement out? Or alternatively is there a way to automate the call to shutdown?

Reply:
Currently this is needed. A better solution from @RyanUnderhill is still WIP.